### ISSUE TITLE
Sydney, Perth?

### ISSUE THREAD

**Issue by stravadk** [JamesChevalier/cities#7]
Just noticing that many cities in Australia are absent, while there are abundances of cities elsewhere.

**Comment by JamesChevalier** [comment]
Yeah, this has to do with the `Admin Level` being different for Australian cities.

I imported the United States first, so I've gone across the other countries expecting Cities to exist at `Admin Level` 8. For example, [Boston](https://nominatim.openstreetmap.org/details.php?place_id=198368368) is `Admin Level` 8, but compare that to [Sydney](https://nominatim.openstreetmap.org/details.php?place_id=198993911) which is `Admin Level` 7.

So I have to look through a lot of the Australian cities, see which `Admin Level` they exist at, and re-run my script to pull _that_ `Admin Level` out of OpenStreetMap instead of 8.
